### PR TITLE
Assignment3

### PR DESCRIPTION
**Pull request by JeffHowes**
Pull request for assignment 3

### CODE REVIEW

File: Assignment3/HW3-Roshambo/Player.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        private int playersPoints;
+        private string playersWeaponChoice;
+        private string playersName;       
+        
+        public Player(string name)
+        {            
+            playersName = name;
+        }
+
+        public void initialPlayersPoint()
```
Comment (breaman) [inline Assignment3/HW3-Roshambo/Player.cs]: You have some camelCase methods and some PascalCase methods. C# uses PascalCase and you will be marked down in the future if you don't use PascalCase for method names. Either way, you should have picked one fashion and stuck with it throughout.

File: Assignment3/HW3-Roshambo/Player.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            else                
+                playersWeaponChoice = "scissors";
+        }
+
+        public string getWeaponPicked()
+        {
+            return playersWeaponChoice;
+        }
+
+        public void SubtractPoints()
```
Comment (breaman) [inline Assignment3/HW3-Roshambo/Player.cs]: There is a logic flaw here. When playing the game, I got this behavior (it was the first move of the game, so we both had 100 to start out with.
```
You picked rock
The computer picked paper
The computer won this round!
You have 80 points and the computer has 100 points
```

File: Assignment3/RoshamboUnitTest/UnitTest1.cs [hunk 3]
```diff
@@ -0,0 +1,266 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+
+namespace RoshamboUnitTest
+{
+    [TestClass]
+    public class UnitTest1
```
Comment (breaman) [inline Assignment3/RoshamboUnitTest/UnitTest1.cs]: This class should be better named than just UnitTest1.

File: Assignment3/RoshamboUnitTest/UnitTest1.cs [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+            // Act            
+            human.validHumansWeaponChoice("Lizzard");
+
+            // Assert
+            Assert.AreEqual("rock", human.getWeaponPicked());
+            
+        }//end of test method
+
+        [TestMethod]
+        public void SubtractPoints_ForRock_100PointsDownTo80()
```
Comment (breaman) [inline Assignment3/RoshamboUnitTest/UnitTest1.cs]: What are the SubtractPoints tests actually testing? You are setting the weapon and then subtracting points from the same player who has the weapon? Shouldn't it subtract 20 from computer if the human won and the human's weapon of choice was a rock?

File: Assignment3/RoshamboUnitTest/UnitTest1.cs (hunk 3 above)
Comment (breaman) [reply Assignment3/RoshamboUnitTest/UnitTest1.cs]: You are missing tests for your AdjustPoints method which would have caught your logic flaw on points being subtracting correctly based on what object the winner used, not what object the loser used.

File: Assignment3/HW3-Roshambo/Player.cs (hunk 2 above)
Comment (JeffHowes) [reply Assignment3/HW3-Roshambo/Player.cs]: Mike, I don't see the logic error here.  Paper bets rock, so the computer should have won as it did and the users points should have been reduced, as they were by 20 points (per the requirements). Can you please explain the bug you are seeing?

**Pull request closed without merging**